lib/nethealth/nethealth.go

```diff
@@ -496,8 +496,9 @@ func (s *Server) processAck(e messageWrapper) error {
 		if err != nil {
 			return trace.Wrap(err)
 		}
-		if pkt.Seq != peer.echoCounter {
-			return trace.BadParameter("Response sequence doesn't match latest request.")
+		if uint16(pkt.Seq) != uint16(peer.echoCounter) {
+			return trace.BadParameter("response sequence doesn't match latest request. Expected %d, received %d",
```

Comment:
Probably should add the variables as fields to the error instead of formatting as a string.

Something along the lines of:
```
return trace.BadParameter("response sequence doesn't match latest request.").AddFields(map[string]interface{}{
    "expected": uint16(peer.echoCounter),
    "received": uint16(peer.echoRequest),
}
```